models/intel/index.md

```diff
@@ -18,74 +17,73 @@ detect the same types of objects (for example, `face-detection-adas-0001` and
-| [person-vehicle-bike-detection-2004](./person-vehicle-bike-detection-2004/README.md)                                                        | 1.811                | 2.327      |       | X       | X        | X     |                |         |
-| [vehicle-license-plate-detection-barrier-0106](./vehicle-license-plate-detection-barrier-0106/README.md)                          | 0.349                | 0.634      |       |         | X        |       | X              |         |
-| [product-detection-0001](./product-detection-0001/README.md)                                                                                            | 3.598                | 3.212      |       |         |          |       |                | X       |
-| [person-detection-asl-0001](./person-detection-asl-0001/README.md)                                                                                   | 0.986                | 1.338      |       |      X  |          |       |                |         |
-| [yolo-v2-ava-0001](./yolo-v2-ava-0001/README.md)                                                                                                              | 29.38                | 48.29      |       |      X  | X        | X     |                |         |
-| [yolo-v2-ava-sparse-35-0001](./yolo-v2-ava-sparse-35-0001/README.md)                                                                                | 29.38                | 48.29      |       |      X  | X        | X     |                |         |
-| [yolo-v2-ava-sparse-70-0001](./yolo-v2-ava-sparse-70-0001/README.md)                                                                                | 29.38                | 48.29      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-ava-0001](./yolo-v2-tiny-ava-0001/README.md)                                                                                               | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-ava-sparse-30-0001](./yolo-v2-tiny-ava-sparse-30-0001/README.md)                                                                 | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-ava-sparse-60-0001](./yolo-v2-tiny-ava-sparse-60-0001/README.md)                                                                 | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-vehicle-detection-0001](./yolo-v2-tiny-vehicle-detection-0001/README.md)                                                     | 5.424                | 11.229     |       |         | X        |       |                |         |
-
+| Model Name | Accuracy | Complexity (GFLOPs) | Size (Mp) |
```

Comment:
"Accuracy" by itself doesn't mean much; it should name the specific metric that is used.

Reply:
accuracy column removed